script.js

```diff
@@ -4,6 +4,8 @@
 /* eslint-disable quotes */
 /* eslint-disable array-callback-return */
 /* eslint-disable no-unused-vars */
+/* eslint-disable linebreak-style */
+

```

Comment:
- [ ] @prg-04 and @Ellon-M its not god practice to disable linters for entire files, kindly fix the linters instead of disabling them.
`NB its slightly allowed to disable linters in some lines of code if you are unable to fix it but not the entire file.`